cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -29,8 +29,10 @@
  {% set feature_gates = "--feature-gates=" + grains.feature_gates -%}
 {% endif -%}
 
+{% set throttles = "--iptables-sync-period=1m --iptables-min-sync-period=10s" -%}
```

Comment:
In general the bursts should allow changes to happen very quickly most of the time, and throttling kicks in only in larger clusters.  At least, that's what I expect...